packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -68,6 +88,17 @@ class Layout extends React.Component {
             />
           </nav>
           <main id="main-container" className={cx('main-container')}>
+            {alerts.alertList.map(
```

Comment:
the alerts object isn't listed as required.

Reply:
I will list it as required in next commit